libsmtutil/SMTLib2Interface.cpp

```diff
@@ -126,7 +126,7 @@ pair<CheckResult, vector<string>> SMTLib2Interface::check(vector<Expression> con
 		result = CheckResult::ERROR;
 
 	vector<string> values;
-	if (result == CheckResult::SATISFIABLE && result != CheckResult::ERROR)
+	if (result == CheckResult::SATISFIABLE && !_expressionsToEvaluate.empty())
```

Comment:
@chriseth this was apparently a bug here that caused the `solc-js` test failure. No model was requested in this test (it uses only literals), and because of the solver response `sat\n`, it was parsing an empty value which caused an assertion failure at `BMC.cpp:767`.